src/app/component/auth/components/restore/restore.component.html

```diff
@@ -1,11 +1,13 @@
 <div>
   <label>{{'user.restore.email' | translate}} </label>
-  <input class="form-control email" name="email" [(ngModel)]="email"
-         minlength="1" email="true">
-  <div style=" margin-top: 40px;text-align:center">
+  <input class="form-control email" 
+         name="email" 
+         [(ngModel)]="email"
+         minlength="1" 
+         email="true">
+  <div style="margin-top: 40px; text-align:center">
```

Comment:
Why have you left inline styles instead of creating scss file or setting the styles property in the component metadata?